README.md

```diff
@@ -39,6 +39,7 @@ This section documents the available CLI parameters for controlling what gets ge
  |   `--http-model-opts`  | Select the options for the http models that you want to be generated.
  |                        | CHOOSE ANY OF:
  |                        |   `JAVA_SERIALIZATION` - This option adds Java Serializable interface to the generated models
+ |                        |   `QUARKUS_REFLECTION_CONFIG` - This options generates the reflection-config.json file for quarkus integration projects
```

Comment:
What about splitting this into two options:
`--http-model-opts=QUARKUS_REFLECTION` This option adds `@RegisterForReflection` to the generated models. Requires dependency "'io.quarkus:quarkus-core:+"

And a separate target for generating the json reflection:
`--targets= QUARKUS_REFLECTION_CONFIG` This options generates the reflection-config.json file for quarkus integration projects

Reply:
👍

Reply:
sold!

Reply:
🥳